frontend/src/views/Sandbox.vue

```diff
@@ -1,10 +1,17 @@
 <template>
-  <div class=""></div>
+  <div class="">
```

Comment:
Should revert Sandbox to as it was prior to this PR - If you want, you could create a new View vue file for the OAuth component, but I believe that is covered by #28

Reply:
I have reverted Sandbox and will allow #28 or subsequent to solidify the OAuth View